netlify/edge-functions/redirect-null.js

```diff
@@ -0,0 +1,22 @@
+export default async (request, context) => {
+  const url = new URL(request.url);
+  if (url.pathname.endsWith('/null') || url.pathname.endsWith('/null/')) {
+    // Remove the "/null" part from the URL
+    let newPathname = url.pathname.replace(/\/null\/?$/, '/');
```

Comment:
you could save the url.pathname to a var to avoid repeating it



```suggestion
const pathname = url.pathname;

  if (pathname.endsWith('/null') || pathname.endsWith('/null/')) {
    // Remove the "/null" part from the URL
    let newPathname = pathname.replace(/\/null\/?$/, '/');
```